src/resolvers/Query/flatRateFulfillmentMethods.js

```diff
@@ -22,13 +22,22 @@ export default async function flatRateFulfillmentMethods(_, args, context, info)
 
   const shopId = decodeShopOpaqueId(opaqueShopId);
 
-  const query = await context.queries.flatRateFulfillmentMethods(context, {
+  const { collection, pipeline } = await context.queries.flatRateFulfillmentMethods(context, {
     shopId
   });
 
-  return getPaginatedResponse(query, connectionArgs, {
+  const paginated = await getPaginatedResponseFromAggregate(collection, pipeline, connectionArgs, {
     includeHasNextPage: wasFieldRequested("pageInfo.hasNextPage", info),
     includeHasPreviousPage: wasFieldRequested("pageInfo.hasPreviousPage", info),
     includeTotalCount: wasFieldRequested("totalCount", info)
   });
+
+  // getPaginatedResponsFromAggregate return null when no record return
+  // workaround by setting to 0 when total count is null
+  const preventZeroTotal = {
+    ...paginated,
+    totalCount: paginated.totalCount || 0
+  }
```

Comment:
The latest version of `api-utils` should fix this, so update that dependency and remove this workaround.